main.development.js

```diff
@@ -47,7 +47,7 @@ if (process.env.NODE_ENV !== 'development') {
 const createWindow = () => {
   mainWindow = new BrowserWindow({
     'show': false,
-    'resizable': false,
+    'resizable': true,
```

Comment:
What is the relevance of this to the PR?

Reply:
No relevance at all. The problem was that I am developing on MacOS and I could not see all the content of the window if it was not resizable. You can leave it as before.